test/unit/batch_alloc.c

```diff
@@ -59,10 +59,28 @@ verify_batch(void **ptrs, size_t batch, size_t usize, bool zero) {
 	}
 }
 
+typedef struct batch_alloc_packet_s batch_alloc_packet_t;
+struct batch_alloc_packet_s {
+	void **ptrs;
+	size_t num;
+	size_t size;
+	int flags;
+};
+
+static size_t
+batch_alloc(void **ptrs, size_t num, size_t size, int flags) {
```

Comment:
I don't think it's necessary now, but I think we'll eventually have to have a version of this API that returns a function pointer to the implementation. The mallctl pathways are pretty slow. But it's fine to wait until we have benchmark data or something, too.

Reply:
Agree on the function pointer part. (Regarding this particular change, I was just adding this convenient function in the test to avoid code duplication.)